src/index.html

```diff
@@ -15,8 +15,93 @@
       rel="stylesheet"
       href="./style.css"
     />
+    <link
+      rel="stylesheet"
+      href="https://fonts.google.com/specimen/Roboto"
+    />
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header>
+      <div class="container">
+        <a
+          class="logo"
+          href="index.html"
+        >
+          <img
+            src="/src/images/logo.png"
+            alt="moyo"
+          />
```

Comment:
The `href` attribute in the <a> tag should not contain spaces. Consider changing `href="index.html"` to `href="#home"` or another appropriate anchor link.